RMS-53692 Добавить в документацию по АПИ инфо о поездке и связанных с…

OlgaBelilovskaya opened the pull request:
… поездкой методах.

Commit:
RMS-53692 Добавить в документацию по АПИ инфо о поездке и связанных с поездкой методах.

zed220 commented:
Лучше каждое предложение делать на новой строке. Так ревьювить проще.

zed220 commented on _posts/2022-08-16-ride.md:
```diff
@@ -0,0 +1,18 @@
+
+В API V8Preview2 добавлено понятие связанной с доставкой поездки [`Ride`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Brd_Ride.htm), а также несколько методов по работе с новой сущностью.
+
+Поездка нужна для работы со внешними курьерскими службами (сейчас это Яндекс-Доставка) для отслеживания пути доставочного заказа от начального статуса поиска внешнего курьера до финального статуса доставки заказа в руки клиенту. По мере изменения свойства [`RideStatus`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Brd_RideStatus.htm) официант может видеть, в какой стадии на ...
```
```suggestion
Поездка используется при работе с внешними курьерскими службами (сейчас это Яндекс-Доставка) для отслеживания пути доставочного заказа от начального статуса поиска внешнего курьера до финального статуса доставки заказа в руки клиенту. По мере изменения свойства [`RideStatus`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Brd_RideStatus.htm) официант может видеть, в какой стадии находится доставка заказа клиенту.
```

zed220 commented on _posts/2022-08-16-ride.md:
```diff
@@ -0,0 +1,18 @@
+
+Поездка нужна для работы со внешними курьерскими службами (сейчас это Яндекс-Доставка) для отслеживания пути доставочного заказа от начального статуса поиска внешнего курьера до финального статуса доставки заказа в руки клиенту. По мере изменения свойства [`RideStatus`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Brd_RideStatus.htm) официант может видеть, в какой стадии на ...
+
+В свойстве [`Ride.ExternalId`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Brd_Ride_ExternalId.htm) хранится идентификатор данного заказа во внешней системе. В свойстве [`Ride.StatusDetails`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Brd_Ride_StatusDetails.htm) из внешней системы можно передать во фронт какую-то нужную для пояснения положения дел  ...
```
```suggestion
В свойстве [`Ride.ExternalId`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Brd_Ride_ExternalId.htm) хранится идентификатор данного заказа во внешней системе. С помощью свойства [`Ride.StatusDetails`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Brd_Ride_StatusDetails.htm) можно передать из внешней системы во фронт дополнительную информацию о поездке.
```

OlgaBelilovskaya committed:
Update _posts/2022-08-16-ride.md

Перефразировала.

Co-authored-by: Zed <pzed220@gmail.com>

OlgaBelilovskaya committed:
Update _posts/2022-08-16-ride.md

Тут тоже перефразировала, как ты предложил.

Co-authored-by: Zed <pzed220@gmail.com>

zed220 commented on _posts/2022-08-16-ride.md:
```diff
@@ -0,0 +1,18 @@
+---
```
Сделай, пожалуйста, каждое предложение на новой строке. Можешь посмотреть, как в других местах это уже сделано.

OlgaBelilovskaya replied:
Сделала, спс. Посмотри пожалуйста.

OlgaBelilovskaya committed:
Update 2022-08-16-ride.md

Каждое предложение с новой строки.

vcpp commented on _posts/2022-08-16-ride.md:
```diff
@@ -0,0 +1,22 @@
+
+В API V8Preview2 добавлено понятие связанной с доставкой поездки [`Ride`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Brd_Ride.htm), а также несколько методов по работе с новой сущностью.
+
+Поездка используется при работе с внешними курьерскими службами (сейчас это Яндекс-Доставка) для отслеживания пути доставочного заказа от начального статуса поиска внешнего курьера до финального статуса доставки заказа в руки клиенту.  
```
Яндекс Доставка без дефиса вроде бы? Мимо брендбука смотрится непривычно.

OlgaBelilovskaya replied:
Убрала дефис.

vcpp commented on _posts/2022-08-16-ride.md:
```diff
@@ -0,0 +1,22 @@
+
+С помощью свойства [`Ride.StatusDetails`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Brd_Ride_StatusDetails.htm) можно передать из внешней системы во фронт дополнительную информацию о поездке.
+
+Для работы с поездками в [`IOperationService`](https://iiko.github.io/front.api.sdk/v8/html/Methods_T_Resto_Front_Api_IOperationService.htm) добавлены три метода.
```
Тут бы двоеточие и дальше список из трёх пунктов.

OlgaBelilovskaya replied:
Сделала список.

OlgaBelilovskaya committed:
Update 2022-08-16-ride.md

Убрала дефис из названия бренда "Яндекс Доставка". Перечисление методов работы с поездкой сделала списком.
